### ISSUE TITLE
artisan resource command - class not imported

### ISSUE THREAD

**Issue by frezno** [laravel/nova-issues#1230]
When creating a resource via artisan the entry `use App\Nova\Resource;` is not included in the template file but has to be added manually. Can't remeber to have this 'issue' with < 1.3.0
Not a bug actually but a nice to have if it would be included.
btw. can't find where this artisan command is handled within the source, otherwise i'd fix it (for me) and/or write a patch request.

**Comment by dillingham** [comment]
https://github.com/laravel/nova/blob/master/src/Console/stubs/resource.stub

**Comment by dillingham** [comment]
This is the same behavior if you did php artisan make:controller 
No `App\Http\Controllers\Controller` is added because it's in the same namespace

**Comment by frezno** [comment]
That's true, @davidhemphill as long as classes don't live in a subdirectory
